common.mk

```diff
@@ -57,6 +57,14 @@ setup: install-uv setup-precommit install-build-deps ## Set up a development env
 setup-tests: install-uv install-build-deps ##- Set up a testing environment without linters
 	uv sync $(UV_TEST_GROUPS)
 
+.PHONY: setup-tics
+setup-tics: install-uv install-build-deps ##- Set up a testing environment for Tiobe TICS
```

Comment:
Can you see us needing any additional (non-python) dependencies for tics stuff? If so, let's make `install-tics-deps` as an empty target in `Makefile` and depend on it here. (If not this is a no-op).

Reply:
Good plan for when we need this, but I cannot think of a reason now